src/components/Compare/CompareTableRow.tsx

```diff
@@ -77,23 +80,32 @@ const CompareTableRow = (props: {
           iconColor={location.metricsInfo.level}
           sortByPopulation={sortByPopulation}
         >
-          <div>
-            <Rank>{location.rank}</Rank>
-            <FiberManualRecordIcon />
-          </div>
-          <div>
-            <StyledRegionName
-              showStateCode={showStateCode}
-              region={location.region}
-              condensed
-            />
-            <br />
-            <LocationInfoWrapper>
-              <Population>
-                {formatEstimate(location.region.population, populationRoundTo)}
-              </Population>
-            </LocationInfoWrapper>
-          </div>
+          <LocationCellWrapper>
+            <LocationRankWrapper>
+              <div>
```

Comment:
we don't need this inner div anymore now that there's a wrapper

Reply:
Oh right, forgot to take those out!